lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,13 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://www.fontanka.ru/24hours_news.html"],
+    "headers": {"user-agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 YaBrowser/24.1.0.0 Safari/537.36",
```

Comment:
apply better formatting to JSON. for example, you can see how it should be formatted: https://jsonformatter.curiousconcept.com/#